tests/toolchain/crash.pm

```diff
@@ -61,18 +61,15 @@ sub run() {
 
     # activate kdump
     script_run 'yast2 kdump', 0;
-    if (check_screen 'yast2-kdump-disabled') {
-        send_key 'alt-u';    # enable kdump
-    }
-
-    assert_screen 'yast2-kdump-enabled';
-    send_key 'alt-o';        # OK
-
-    if (check_screen 'yast2-kdump-restart-info') {
-        send_key 'alt-o';    # OK
-    }
-
-    wait_still_screen(10, 30);
+    my @tags = qw(yast2-kdump-disabled yast2-kdump-enabled yast2-kdump-restart-info yast2-missing_package yast2_console-finished);
+    do {
+        assert_screen \@tags, 300;
```

Comment:
Previously it was adding up all the (wasted) wait times for check screens. Now, we wait up to 300 but for any in the list. At least 30 is not enough what my tests confirmed.